File: FE/FE_Client/src/models/dto/checkout/CheckoutInfoDto.ts
import { AddressInfoDto } from './AddressInfoDto';
import { ShippingRespondDto } from './ShippingRespondDto';
import CartItem from '../../CartItem';
export interface CheckoutInfoDto {
    addressInfoDto: AddressInfoDto;
    cartProductDtoList: CartItem[];
    shippingRespondDtoList: ShippingRespondDto[];
}

File: BE/WebParent/BE_Client/src/main/java/com/pbl5/client/dto/checkout/CheckoutInfoDto.java
package com.pbl5.client.dto.checkout;

import com.pbl5.client.dto.CartProductDto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class CheckoutInfoDto {

    private AddressInfoDto addressInfoDto;
    private List<CartProductDto> cartProductDtoList;
    private List<ShippingRequestDto> shippingRequestDtoList;
    private List<ShippingRespondDto> shippingRespondDtoList;

}
